lightning/src/ln/peer_handler.rs

```diff
@@ -1043,6 +1043,7 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref> PeerManager<D
 	fn forward_broadcast_msg(&self, peers: &mut PeerHolder<Descriptor>, msg: &wire::Message, except_node: Option<&PublicKey>) {
 		match msg {
 			wire::Message::ChannelAnnouncement(ref msg) => {
+				log_trace!(self.logger, "Sending message {:?} to all peers except {:?} or its sources", msg, except_node);
```

Comment:
Hmm, but we're sending the message to everything excpet (except_node or message source)?

Reply:
Ah, not sure I understood. So it's not being sent to the peer that sent the message to us (i.e., `except_node`) if applicable) and the channel counterparties (i.e., "its sources"). Maybe replace "its sources" with "the announced channel's counterparties".

As for "and" vs "or", in some cases it could be both (e.g., our peer sends a `ChannelAnnouncement` for a channel where we are peers with one or more of the channel's counterparty. Maybe and/or would be appropriate? Maybe it's just pedantic. 😛